### DISCUSSION TITLE
About using SSL certificate in wiki.js

### DISCUSSION THREAD

**Discussion by fandd1234** [requarks/wiki#6220] [Help / Questions]
Probably a stupid question but I've been struggling to configure the SSL cert for my wiki.js quite a while. :)

Here's my scenario:
I do have my own SSL cert (pfx and cer) with a provider (eg. godaddy) for my domain (eg. abcd.com)

So I am wondering if I can use letsencrypt to issue the cert for abcd.com and achieve my goal to use SSL cert in wiki.js ?
Is there any conflict using between them?

Thank you.

**Comment by s0p4L1n3** [comment]
Here's an answer
(https://comodosslstore.com/resources/can-you-have-multiple-ssl-certificates-for-one-domain/)

**Reply by s0p4L1n3** [reply]
Can you confirm you are using a reverse proxy to serve WikiJS like NGINX ?

**Reply by fandd1234** [reply]
@s0p4L1n3 I am not using any reverse proxy (apache/NGINX) to serve my WikiJS.  So sounds like letsencrypt is not the best option in my scenario?

Or I need to configure the pfx cert in config.yml just like below screenshot?  Actually I tried to use below config but whenever I enable it, the wiki.js site would be gone down. :(

![image](https://user-images.githubusercontent.com/92530680/222683899-c8520af0-09f8-4636-ae8a-73ad78de7cbd.png)

**Reply by ErixWong** [reply]
Yes I got the same issue when using pfx certificate.
Now I'm using pem certificates.

I'm using frp as reverse proxy.
